### PR TITLE
111-add-code-coverage-support

### LINKED ISSUE #111: Add code coverage support
## Background
Add code coverage support to be able measure code quality.

## Feature
New ability to measure current code coverage as one of QA metrics.

### PR DESCRIPTION
**Pull request by miroslavpojer**
- new plugin dependency on jacoco library
- new code coverage profile in pom file
- how to section in readme file.

Closes #111

### CODE REVIEW

**Review by wajda — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ mvn clean install -Pscala-2.13
 ./build-all.sh
 ```
 
+### Measuring code coverage
+```shell
+./mvn clean verify -Pcode-coverage
+```
+Code coverage will be generated on path:
+```aidl
```
Comment (miroslavpojer) [inline README.md:40]:
```suggestion
```
```

File: README.md [hunk 2]
```diff
@@ -37,7 +37,6 @@ mvn clean install -Pscala-2.13
 ./mvn clean verify -Pcode-coverage
 ```
 Code coverage will be generated on path:
-```aidl
```
Comment (wajda) [inline README.md]: Why?
The backticks are now imbalanced after this change....

Comment (miroslavpojer) [reply README.md]: It was unsuccessful try to remove wrong tag using suggestion feature. It failed. Problem has been solved in commit 9cdca671a9d28249f6d958bdf41df997d8f6b1ee.

**Review by wajda — APPROVED** [review]

**Review by benedeki — APPROVED** [review]

**Pull request merged**